flixel/input/gamepad/ButtonIndex.hx

```diff
@@ -0,0 +1,399 @@
+package flixel.input.gamepad;
+import flixel.input.gamepad.FlxGamepadButtonID;
+import flixel.input.gamepad.FlxGamepad.FlxGamepadAnalogStick;
+import flixel.input.gamepad.FlxGamepad.GamepadModel;
+
+/**
+ * ...
+ * @author larsiusprime
+ */
+class ButtonIndex
```

Comment:
Yeah works for me. FlxGamepadMap is probably no good b/c that suggests it's an actual Map type, I like "Mapping"